src/templates/flow.json

```diff
@@ -44,7 +44,7 @@
   "accounts": {
     "emulator-account": {
       "address": "f8d6e0586b0a20c7",
-      "key": "4d9287571c8bff7482ffc27ef68d5b4990f9bd009a1e9fa812aae08ba167d57f"
+      "key": "479a31b4171cd052671adb8f5125ffdabe00d23dafa218e8ae77a432590f21e7"
```

Comment:
Emulator service account private key was changed?

Reply:
Hmm, I don't think so. I had to switch back to the old key to get it working. 🤔

Reply:
```suggestion
      "key": "4d9287571c8bff7482ffc27ef68d5b4990f9bd009a1e9fa812aae08ba167d57f"
```